unicorecmsgem/templates/404.pt

```diff
@@ -0,0 +1,18 @@
+<div id="content" 
+    metal:use-macro="view.global_template" 
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    i18n:domain="unicorecmsgem">
+    <div metal:fill-slot="search"></div>
+    <div class="page-not-found" metal:fill-slot="content">
```

Comment:
Rather than add a new class, here re-use an existing one. Try `.articles .detail` to get the padding we want.